nbgitpuller/plugins/test_gdrive.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import print_function
```

Comment:
this file should be moved to /tests, shouldn't be inside the package

Reply:
fixed in latest commit

Reply:
doesn't appear to be moved in the latest commit @ritvik-iyer ?

Reply:
fixed.